### PR TITLE
Add backward_warping to digital_image_processing

### PR DESCRIPTION
**Pull request by TimelessPersona**
### Describe your change:

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by abhishekmulik** [comment]
Looks good !! nice work

**Review by glass2300 — COMMENTED** [review]
looks good, nice

File: digital_image_processing/backward_warping/backward_warping.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    angle = np.deg2rad(15)
+    M3 = np.array(
+        [
+            [np.cos(angle), -np.sin(angle), 0],
+            [np.sin(angle), np.cos(angle), 0],
+            [0, 0, 1],
+        ]
+    )
+
+    M4 = np.array([[1, 0.2, 0], [0.2, 1, 0], [0, 0, 1]])
```
Comment (CaedenPH) [inline digital_image_processing/backward_warping/backward_warping.py:96]:
```suggestion
    M1 = np.array([[1, 0, 50], [0, 1, 100], [0, 0, 1]])
    M2 = np.array([[1.5, 0, 0], [0, 1.5, 0], [0, 0, 1]])
    M3 = np.array(
        [
            [np.cos(angle), -np.sin(angle), 0],
            [np.sin(angle), np.cos(angle), 0],
            [0, 0, 1],
        ]
    )
    M4 = np.array([[1, 0.2, 0], [0.2, 1, 0], [0, 0, 1]])
    
    angle = np.deg2rad(15)
```

**Review by CaedenPH — CHANGES_REQUESTED** [review]
Please could you add some [doctests](https://docs.python.org/3/library/doctest.html)

**Comment by tianyizheng02** [comment]
Closing due to lack of doctests and lack of response

**Pull request closed without merging**